vspace/src/main/webapp/WEB-INF/views/staff/modules/sequences/sequence.jsp

```diff
@@ -1,23 +1,200 @@
-<%@ page pageEncoding="UTF-8" %>
+<%@ page pageEncoding="UTF-8"%>
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
-<%@ taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn"%>
 <%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
 
 <ol class="breadcrumb">
-  <li class="breadcrumb-item"><a href="<c:url value="/staff/dashboard" />">Dashboard</a></li>
-  <li class="breadcrumb-item"><a href="<c:url value="/staff/module/list" />">Modules</a></li>
-  <li class="breadcrumb-item"><a href="<c:url value="/staff/module/${module.id}" />">${module.name}</a></li>
-  <li class="breadcrumb-item active">${sequence.name}</li>
+	<li class="breadcrumb-item"><a
+		href="<c:url value="/staff/dashboard" />">Dashboard</a></li>
+	<li class="breadcrumb-item"><a
+		href="<c:url value="/staff/module/list" />">Modules</a></li>
+	<li class="breadcrumb-item"><a
+		href="<c:url value="/staff/module/${module.id}" />">${module.name}</a></li>
+	<li class="breadcrumb-item active">${sequence.name}</li>
 </ol>
 
-<h1>Sequence: ${sequence.name}</h1>
+
+
+<script>
+
+
+$(document).ready(function() { 
+ 
+	
+    //-------- edit contentblock description Sequence starts --------
+    $("#submitDescription").hide()
+    $("#cancelEditDescription").hide()	
+    
+    $("#editDescription").click(function() {
+    	var description = $("#description").text()
+    	document.getElementById("description").value = description;
```

Comment:
we're using jquery. let's not mix jquery and plain js unless we have to.

Reply:
also, why do you set the value field of the description element?

Reply:
In the code, value of the description is getting removed. Hence, I stored it in value attribute so that it can be retrieved later. Otherwise, it would be difficult to fetch original value.

Reply:
I don't think it's proper HTML to give p elements a value attribute. Use data attributes instead.